core/src/test/scala/akka/persistence/cassandra/CassandraSpec.scala

```diff
@@ -159,7 +159,7 @@ abstract class CassandraSpec(
         override def write(b: Int): Unit = oldOut.write(b)
       }) {
         override def println(x: Any): Unit =
-          oldOut.println(prefix + String.valueOf(x).replaceAllLiterally("\n", s"\n$prefix"))
+          oldOut.println(prefix + String.valueOf(x).replace("\n", s"\n$prefix"))
```

Comment:
This is not the same, it replaces `\n` with the corresponding position in the second `CharSeq` param, so `\n`. I think it should be `String#replaceAll` rather.

Reply:
It should be the same as JavaDoc of `replaceAllLiterally` states 
```
Replace all literal occurrences of literal with the literal string replacement. This method is equivalent to #replace.
```

The difference between `replace` and `replaceAll` is that the latter is using a regex.

Reply:
Ah, it is indeed me parsing the JDK javadoc wrong. 👍